index.html

```diff
@@ -0,0 +1,286 @@
+                    <a class="menuformat" href="#">Deals</a>
+                    <a class="menuformat" href="#">TNMW Porgrams</a>
+                </div>
+            </div> 
+            <div class="entiremenu">
+                <div class="topmenu">
+                    <ul class="topmenul">
+                        <li class="news"><a href="#">News</a></li>
+                        <li class="eventdropdown">
+                            <a class="eventsbtn" href="#">Events ▾</a>
+                            <div class="eventscontent">
+                                <a class="menuformat" href="">TNW2020</a>
+                                <a class="menuformat" href="">Couch Conferences</a>
+                            </div>
+                        </li>
+                        <li class="eventdropdown">
+                            <a class="eventsbtn" href="#">Business ▾</a>
+                            <div class="eventscontent">
+                                <a class="menuformat" href="">Index</a>
+                                <a class="menuformat" href="">Programs</a>
+                            </div>
+                        </li>
+                        <li><a href="#">AMAs</a></li>
+                        <li><a href="#">Spaces</a></li>
+                        <li><a href="#">Term & Conditions</a></li>
+                    </ul>
+                    <ul class="topmenur">
+                        <li><a href="#">About</a></li>
+                        <li><a href="#">Advertise</a></li>
+                        <li><a href="#">Jobs</a></li>
+                        <li><a href="#">Contact</a></li>
+                    </ul>
+                </div>
+                <div class="menuicon">
+                    <div class="centermenu">
+                        <a href="#" class="navigation">LATEST</a>
+                        <a href="#" class="navigation">HARD FORK</a>
+                        <a href="#" class="navigation">PLUGED</a>
+                        <a href="#" class="navigation">README</a>
+                        <a href="#" class="navigation">GROWTH QUARTERS</a>
+                    </div>
+                    <div class="nvbutton">
+                        <a class="toprbutton" href="#"><img src="https://cdn3.iconfinder.com/data/icons/flaticons-1/24/flaticon_search-512.png" alt="" width="20"></a>
+                        <a id="navigationmenu" class="toprbutton" href="#"><img src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/b2/Hamburger_icon.svg/1200px-Hamburger_icon.svg.png" alt="" width="20"></a>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <div class="nvaddbar">
+            <a href="#">Calling young digital talent in the Netherlands – Be recognized in the T500 →</a>
+        </div>
+    </header>
+    <section class="highlights">
+        <h2 class="feature-logo">Tnw</h2>
+        <div class="main-feature divs">
```

Comment:
This `div` tag should be an `article` tag instead. DO this for all highlights-articles 🔴